### ISSUE TITLE
Add * and ** to args and kwargs in rendered function signature

### ISSUE THREAD

**Issue by machow** [machow/quartodoc#68]
<img width="842" alt="image" src="https://user-images.githubusercontent.com/2574498/225719241-d27aa82e-f913-4390-a225-6b9f30622578.png">

<img width="802" alt="image" src="https://user-images.githubusercontent.com/2574498/225719361-8631db4d-71d4-4200-8dda-019c47cd2563.png">

**Comment by machow** [comment]
```
from quartodoc import get_object, preview

# has a * to indicate subsequent args are keyword only
# note that we might infer this when a parameter.kind switches
# to keyword only
obj = get_object("shiny.ui:input_numeric")
preview(obj.parameters)
```

Here's an example for this function:

```python
def f(x, *args, **kwargs):
    ...
```

```
█─Parameters
├─0 = █─Parameter
│     ├─annotation = None
│     ├─kind = <ParameterKind.positional_or_keyword: 'positional  ...
│     ├─name = 'x'
│     └─default = None
├─1 = █─Parameter
│     ├─annotation = None
│     ├─kind = <ParameterKind.var_positional: 'variadic positiona ...
│     ├─name = 'args'
│     └─default = '()'
└─2 = █─Parameter
      ├─annotation = None
      ├─kind = <ParameterKind.var_keyword: 'variadic keyword'>
      ├─name = 'kwargs'
      └─default = '{}'
```

**Comment by machow** [comment]
Fixed by #64

**Comment by machow** [comment]
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/2574498/227377887-be8254e3-cccf-4138-84f7-74ea9920431c.png">
